core/src/main/scala/com/nvidia/spark/rapids/tool/tuning/QualificationAutoTuner.scala

```diff
@@ -29,7 +31,16 @@ class QualificationAutoTuner(
     platform: Platform,
     driverInfoProvider: DriverLogInfoProvider)
   extends AutoTuner(clusterProps, appInfoProvider, platform, driverInfoProvider,
-    QualificationAutoTunerConfigsProvider)
+    QualificationAutoTunerConfigsProvider) {
+
+  /**
+   * List of recommendations for which the Qualification AutoTuner skips calculations and only
+   * depend on default values.
+   */
+  override protected val limitedLogicRecommendations: mutable.HashSet[String] = mutable.HashSet(
```

Comment:
QQ:
It is expected that `limitedLogicRecommendations` to be platform specific.
Is it going to be feasible to do change behavior based on platform when needed in the future?

Reply:
Yes, we could use a similar class based approach (say `PlatformSpecificAutoTunerProvider`) to provide any platform specific tunings.